.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+	"presets": [
+		"@babel/preset-env",
+		"@babel/preset-react",
+		"@babel/preset-typescript"
+	]
+}
```

Comment:
I'd vote for deleting this file if we're not using it now

Reply:
required for unit tests